virtool/tasks/main.py

```diff
@@ -86,4 +88,8 @@ async def create_task_runner_app(config: TaskRunnerConfig):
 
 def run_task_runner(config: TaskRunnerConfig):
     app = create_task_runner_app(config)
-    aiohttp.web.run_app(app=app, host=config.host, port=config.port)
+
+    with suppress(asyncio.CancelledError):
```

Comment:
I don't think suppressing errors here should be required. I suspect it is just hiding another error that should be dealt with more properly. Also I would be hesitant to suppress asyncio cancelled errors without good reason, as it can prevent the task from dying like it should